source/hardware/controllers/gemini_g4v.rst

```diff
@@ -0,0 +1,247 @@
+Gemini G4V
+=====================
+
+.. figure:: ../../_static/controllers/gemini_g4v_topview.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
+
+The Gemini G4V is a 2 deck controller that supports 4 virtual decks, a 4 way mixer and has a built-in 4 channel USB soundcard.
+It includes a Microphone input line, an Aux input line and an Booth output line. This lines are not going through the soundcard but mixed directly at the controller.
+They are not accessible nor controlled by Mixx so they will not appear in any recording nor is possible to control them via the Mixxx screen interface.
+
+
+-  `Manufacturer’s product page <http://geminisound.com/product/g4v>`__
+-  `Forum thread <https://www.mixxx.org/forums/viewtopic.php?f=6&t=12919>`__
+-  `Manufacturer’s User Guide <https://www.manualslib.com/manual/826563/Gemini-G4v.html>`__
+
+.. versionadded:: 2.3
+
+Mixxx sound hardware preferences
+--------------------------------
+
+========== =============
+Output     Channel
+========== =============
+Master     Channel 1 - 2
+Headphones Channel 3 - 4
+========== =============
+
+Mapping Description
+-------------------
+
+Most of the board's controls work as described in the G4V's manual, with a few exceptions. Snapshots of the board's labeling from the Manual are included below:
+
+
+.. figure:: ../../_static/controllers/gemini_g4v_controls.png
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
+
+.. figure:: ../../_static/controllers/gemini_g4v_frontview.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
+
+.. figure:: ../../_static/controllers/gemini_g4v_controllist.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
```

Comment:
This should be text, so it can be read out by a screen reader.

I suggest to use multiple markdown tables, one for each section (top, front, rear).

Reply:
> This should be text, so it can be read out by a screen reader.

Sorry. I don't understand that one. What should be text?

Reply:
OK. Now I got it.
Anyway, not sure how useful is to have that in text, this list is to identify the different controls in the drawing. Any person with impaired vision will have problems to use that information in the images.

Reply:
But it's not searchable with CTRL+F. And rasterized images have bad quality, especially if it's resized for display.

It's also hard to find the control you're looking for, because all controls are in a single list. Using different subsections for top, front, rear would be helpful: Have a look at this manual page for reference: https://manual.mixxx.org/2.3/en/hardware/controllers/roland_dj_505.html#deck-section-p-5-6